application/templates/secretProviderClass.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.secretProviderClass.enabled }}
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: {{ .Values.secretProviderClass.name }}
+  namespace: {{ .Values.secretProviderClass.namespace }}
```

Comment:
namespace shoud be {{ .Release.Namespace }}

Reply:
resolved